ticket_manager_IF.cpp

```diff
@@ -0,0 +1,8 @@
+class ticket_manager_IF
```

Comment:
Proper class names !! 
TicketMangrIF

Do same kind of notation for every file